### PR TITLE
[16.0][IMP] account_cutoff_base: auto reverse

### PR DESCRIPTION
**Pull request by jbaudoux**
Allow to automatically reverse generated moves

Added test move creation

cc @alexis-via @rousseldenis

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by jbaudoux** [comment]
@OCA/accounting-maintainers Can this one be merged ?

**Comment by jbaudoux** [comment]
@OCA/accounting-maintainers Could you merge this PR please ?

**Comment by grindtildeath** [comment]
@jbaudoux it's missing an approval but I'll remove the stale label

File: account_cutoff_base/models/account_cutoff.py
Comment (sbidoul) [inline account_cutoff_base/models/account_cutoff.py:122]: Sorry to be late to the party, but could we make this a computed field based on `env["account.move"].search[("reversed_entry_id", "=", self.move_id)]`.

Comment (sbidoul) [reply account_cutoff_base/models/account_cutoff.py:122]: Or `self.move_id.reversal_move_id and self.move_id.reversal_move_id[0]`

Comment (jbaudoux) [reply account_cutoff_base/models/account_cutoff.py:122]: Sounds better indeed

Comment (jbaudoux) [reply account_cutoff_base/models/account_cutoff.py:122]: I updated to use a compute. Can you re-review? Thanks

**Comment by jbaudoux** [comment]
@sbidoul Can we merge this now?

**Comment by lmignon** [comment]
@sbidoul Can we move forward with this? It works in production for a long time now.

**Comment by sbidoul** [comment]
Yes, I had lost track of this.

/ocabot merge minor

**Pull request merged**
